conans/client/remote_manager.py

```diff
@@ -87,6 +88,8 @@ def get_recipe(self, ref, remote):
         duration = time.time() - t1
```

Comment:
Maybe we could call around here to `get_recipe_snapshot` and store directly (including the sources), we wouldn't need to calculate the checksums locally. It would be 1 more api call... but probably we can live with it.